.github/workflows/create-release-pr-to-develop.yml

```diff
@@ -30,6 +19,17 @@ jobs:
           pr_label: "auto-release-pr-to-develop"
           pr_allow_empty: true
           github_token: ${{ secrets.MERGE_TOKEN }}
+      - uses: actions/checkout@v3
```

Comment:
is this working ? you won't have access to the PR template if you don't checkout before

Reply:
Resolved !